shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/model/Survey.kt

```diff
@@ -1,8 +1,10 @@
 package co.nimblehq.ic.kmm.suv.domain.model
 
 data class Survey(
+    val id: String,
     val title: String,
     val description: String,
     val isActive: Boolean,
-    val coverImageUrl: String
+    val coverImageUrl: String,
+    val questions: List<Question>? = null
```

Comment:
Should we have a `sortedQuestions` getter here to get all questions of a survey sorted by question.displayOrder?

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/113/commits/f771a28f0b85fecfc91063c1c64677166476cd53